omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/OrderGroupResource1_12.java

```diff
@@ -155,18 +155,16 @@ public Model getGETModel(Representation rep) {
 	
 	@Override
 	public Model getCREATEModel(Representation representation) {
-		ModelImpl model = new ModelImpl().property("patient", new StringProperty().example("uuid"))
+		return new ModelImpl().property("patient", new StringProperty().example("uuid"))
 		        .property("encounter", new StringProperty().example("uuid"))
 		        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")))
 		        .property("orderSet", new StringProperty().example("uuid"));
-		return model;
 	}
 	
 	@Override
 	public Model getUPDATEModel(Representation rep) {
-		ModelImpl model = new ModelImpl().property("orders",
+		return new ModelImpl().property("orders",
 		    new ArrayProperty(new RefProperty("#/definitions/OrderCreate")));
```

Comment:
```suggestion
		return new ModelImpl().property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")));
```
Why not in one line?